README.md

```diff
@@ -135,6 +135,23 @@ And you can supply screenshots by creating directories with the following names,
 
 Note that these will replace the current images and screenshots on the play store listing, not add to them.
 
+## Changelogs (What's new)
+
+You can add changelog files under the `changelogs/` directory for each language. The filename should exactly match the version code of the apk that it represents. `supply init` will populate these files from existing Google Play if no metadata exists. 
```

Comment:
:white_check_mark: on the first 3 comments.

"`supply init` will only run if there is no `metadata` folder. Changelogs will be downloaded only when supply init is run." Not sure how to best capture that here.